src/awst_build/eb/bytes-expression-builder.ts

```diff
@@ -193,6 +193,12 @@ export class BytesExpressionBuilder extends InstanceExpressionBuilder<InstanceTy
         )
       case 'bitwiseInvert':
         return new BytesInvertBuilder(this._expr)
+      case 'bitwiseAnd':
+        return new BitwiseAndExpressionBuilder(this._expr)
+      case 'bitwiseOr':
+        return new BitwiseOrExpressionBuilder(this._expr)
+      case 'bitwiseXor':
+        return new BitwiseXorExpressionBuilder(this._expr)
```

Comment:
These could all use the same expression builder (ie. `BitwiseOpBuilder`) and you can pass in the expression + the desired `BytesBinaryOperator`

Reply:
It has been refactored. 898cd45